todo/controllers/task.go

```diff
@@ -5,7 +5,7 @@ import (
 	"strconv"
 
 	"github.com/astaxie/beego"
-	"github.com/beego/samples/todo/models"
+	"samples/todo/models"
```

Comment:
as described in https://github.com/bernardpeh/samples/blob/develop/README.md

i suggest taking samples out from github.com/beego. users can simply clone their own samples repo under the src folder.